lib/fluent/plugin/filter_elasticsearch_timestamp_check.rb

```diff
@@ -11,6 +13,15 @@ def configure(conf)
       require 'date'
       raise Fluent::ConfigError, "specify 1 or bigger number." if subsecond_precision < 1
```

Comment:
This would accept any number larger than 1, however valid options are 3, 6 & 9

Maybe the test here should be more specific?

```
%N Fractional seconds digits, default is 9 digits (nanosecond)
%3N millisecond (3 digits)
%6N microsecond (6 digits)
%9N nanosecond (9 digits)
```

http://ruby-doc.org/stdlib-2.1.1/libdoc/time/rdoc/Time.html

Reply:
Ruby can handle 9 or more precision.
Because Ruby also handles subseconds with `Rational` class.
I think that they are meaningless for now, but they are also correct format in Ruby world.

```
irb> require 'time'
=> true
irb> Time.utc(2000,1,1,0,0,Rational(1,10**12)).strftime("%F %T.%12N")
=> "2000-01-01 00:00:00.000000000001" # pico, 10^-12
irb> Time.utc(2000,1,1,0,0,Rational(1,10**15)).strftime("%F %T.%15N")
=> "2000-01-01 00:00:00.000000000000001" # femto 10^-15
irb> Time.utc(2000,1,1,0,0,Rational(1,10**18)).strftime("%F %T.%18N")
=> "2000-01-01 00:00:00.000000000000000001" # atto 10^-18
```

Reply:
👍 

But maybe worth mentioning in the Readme that common values are:
3 = millisecond
6 = microsecond
9 = nanosecond

Reply:
:+1:
I've added them.